grails-app/conf/Config.groovy

```diff
@@ -132,6 +132,12 @@ grails.validateable.packages = [
         'org.pih.warehouse.shipment',
 ]
 
+org.quartz.scheduler.instanceName = 'DefaultQuartzPool'
+org.quartz.scheduler.instanceId = 'Config.groovy'
```

Comment:
need clarification on instanceId

Reply:
thanks for pointing this out -- see big comment below